examples/gdpr-cookie-consent/README.md

```diff
@@ -0,0 +1,23 @@
+# GDPR Cookie Consent
+
+Create a simple GDPR consent form.
+Till the user doesn't click on the `Accept` button, she will see a banner prompting to accept cookies.
+
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/gdpr-cookie-consent)
+
+## Example
+
+Users will be presented with a GDPR consent form on every page ([app/routes/index.tsx](app/routes/index.tsx)) till they submit the accept button.
```

Comment:
If the form is in the index and not the root, then it’s not going to be on every page